api/Window.json

```
@@ -7715,45 +7715,141 @@
             "deprecated": false
           }
         },
-        "options_parameter": {
+        "options_behavior_parameter": {
           "__compat": {
-            "description": "<code>options</code> parameter",
+            "description": "<code>options.behavior</code> parameter",
             "support": {
               "chrome": {
-                "version_added": "45"
+                "version_added": "35"
               },
               "chrome_android": {
-                "version_added": "45"
+                "version_added": "35"
               },
               "edge": {
                 "version_added": "79"
               },
               "firefox": {
-                "version_added": true
+                "version_added": "27"
```

Based on http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=9636 all of these APIs were introduced in Firefox 36:

```suggestion
                "version_added": "36"
```